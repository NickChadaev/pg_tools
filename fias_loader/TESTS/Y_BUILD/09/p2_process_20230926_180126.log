2023-09-26 18:01:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_09 postgres /home/n.chadaev@abrr.local/Y_BUILD/09/stage_gar_c_09.csv 223-09-25
--------------------------------------------------------
[18:01:26] ... Карачаево-Черкесская Респ, id_region: 10, Execute: /home/n.chadaev@abrr.local/Y_BUILD/09/stage_gar_c_09.csv, process_id:5456
[18:01:26] ... Start process
[18:01:26] ...  -- Version
[18:01:26] ...  -- Очистка от данных
[18:01:26] ...  -- Set UNLOGGED
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_APARTMENT_TYPES_20230925_dec7f8ad-4238-4484-bf99-113a09c762f5.XML
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDR_OBJ_TYPES_20230925_4f714402-bdf0-4b03-8e2f-cda2704fb403.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ROOM_TYPES_20230925_6fedd268-7191-415b-be0e-8a5a110f7099.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OPERATION_TYPES_20230925_d65578ac-f625-4e50-ac1b-06ee9ac4de7a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_PARAM_TYPES_20230925_72393625-9278-4105-b37d-79dc852810ae.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_HOUSE_TYPES_20230925_1bdc09e7-b8f0-40d5-80f3-cf003785714b.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDHOUSE_TYPES_20230925_cf9caa45-fb3f-40d3-8ddf-770f536de4ed.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OBJECT_LEVELS_20230925_8ba71762-ad23-4052-b0b8-a659eac50bbb.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_TYPES_20230925_acf1cb3d-a66a-4e4b-b233-cc9c4032d98a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_KINDS_20230925_a8475f50-3b47-4725-9f46-b1bd99ea576c.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_REESTR_OBJECTS_20230925_53d2b285-3ec7-4199-add7-bda67e703cfe.XML
[18:03:44] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_ADDR_OBJ_20230925_f90ad070-fd53-48c8-8e98-99f7ef9e51dc.XML
[18:03:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_HOUSES_20230925_fd70cbd7-e177-4397-b6c5-a65514f90dce.XML
[18:07:47] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_STEADS_20230925_c040dd55-7c4f-4d7a-b51b-abdf5a64deba.XML
[18:08:12] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_APARTMENTS_20230925_e2dc89a3-4946-4fc3-be2d-9c8694a35178.XML
[18:08:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_ROOMS_20230925_ff06335a-e6f5-4140-8a2a-bd426034496c.XML
[18:08:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_CARPLACES_20230925_5dda6242-c8a2-4061-a27f-6e0e352f0b2e.XML
[18:08:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_ADM_HIERARCHY_20230925_9c87fab6-f931-4956-91b5-f476daf3688a.XML
[18:11:41] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_MUN_HIERARCHY_20230925_2160218c-4469-4887-8e97-90f59247b28b.XML
[18:14:20] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_CHANGE_HISTORY_20230925_bce165f8-529d-4d18-a232-3fa4c4ddc9e8.XML
[18:23:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_ADDR_OBJ_DIVISION_20230925_248bd847-f280-4ab0-91f5-71ecb3778eec.XML
[18:23:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_ADDR_OBJ_PARAMS_20230925_07265883-b9e9-45d6-8110-47934d0dc6e4.XML
[18:24:43] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_STEADS_PARAMS_20230925_91f2614c-3f61-4090-a34c-109171eb1d93.XML
[18:28:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_HOUSES_PARAMS_20230925_ad8466e2-d18f-4130-97b4-c8499faa8256.XML
[18:50:41] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_APARTMENTS_PARAMS_20230925_5f1ca378-932b-47bc-a9ed-2d97388bcf3e.XML
[18:52:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_ROOMS_PARAMS_20230925_469942ba-3d24-4a01-8b7d-6f9557c3d944.XML
[18:52:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_CARPLACES_PARAMS_20230925_d7c45f68-f7c6-4350-91ef-2addaab972da.XML
[18:52:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/09/AS_NORMATIVE_DOCS_20230925_99145b94-22c1-4d81-b7a9-88be49df9aa8.XML
[18:52:50] ...  -- Data finish
[18:52:50] ...  -- Set LOGGED
[18:53:00] ... Stop process
--------------------------------------------------------
