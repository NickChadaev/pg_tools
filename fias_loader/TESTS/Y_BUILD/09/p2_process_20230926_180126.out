 save_gar_version 
------------------
               70
(1 row)

CALL
CALL
 set_gar_dt_create 
-------------------
                70
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 201810
gar_fias.as_addr_obj: 5639
gar_fias.as_houses: 291451
gar_fias.as_steads: 31026
gar_fias.as_apartments: 49608
gar_fias.as_rooms: 88
gar_fias.as_carplaces: 53
gar_fias.as_adm_hierarchy: 218301
gar_fias.as_mun_hierarchy: 209135
gar_fias.as_change_history: 787476
gar_fias.as_addr_obj_division: 83
gar_fias.as_addr_obj_params: 74746
gar_fias.as_steads_params: 348312
gar_fias.as_houses_params: 1782915
gar_fias.as_apartments_params: 131194
gar_fias.as_rooms_params: 118
gar_fias.as_carplaces_params: 0
gar_fias.as_normative_docs: 40986
